internal/service/finspace/kx_cluster_test.go

```diff
@@ -790,7 +1018,7 @@ resource "aws_finspace_kx_cluster" "test" {
 `, rName))
 }
 
-func testAccKxClusterConfig_cacheConfigurations(rName string) string {
+func testAccKxClusterConfig_cacheConfigurations(rName, clusterType string) string {
```

Comment:
Smart idea to try to reuse code here, though in this case we might actually want to keep the config separate, both since we're keeping these tests separate from the base/core acceptance tests, and for clarity. For RDB this is a configuration that should fail, so we might want a separate configuration with a name like rdbCacheConfigurationsNoSavedown

Reply:
Right, that makes sense I'll make the change!